plugin/methodWrappers.go

```diff
@@ -153,6 +162,8 @@ func Read(ctx context.Context, e Entry, size int64, offset int64) (data []byte,
 			size = contentSize
 			err = io.EOF
 		}
+	} else if ReadAction().signature(e) == blockReadableSignature {
+		activity.Warnf(ctx, "size attribute not set for block-readable entry %v", e.id())
```

Comment:
A warning seems fine since this will likely come up when the plugin author's testing their plugin.